lib/players/human.rb

```diff
@@ -0,0 +1,6 @@
+class Human < Player
+  
+  def move(input=nil)
+    input = gets.strip
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(input=nil)
  gets.strip
end